### PR TITLE
1775 add query service

### PR DESCRIPTION
**Pull request by constantin-ciobotaru**
- add feign query service
- add step that use query service
- add @Then check that use the step above

### CODE REVIEW

**Comment by constantin-ciobotaru** [comment]
@Salaboy I've adapted the Diego's code to the last merged code.
@diegonayalazo , please, take a look. Thanks.

File: src/main/resources/stories/runtime-bundle/task-actions.story [hunk 1]
```diff
@@ -11,4 +11,5 @@ When the user starts a random process
 And the user claims a task
 And the user completes the task
 Then the status of the process is changed to completed
+And the status of the process is changed to completed by querying
```
Comment (ffazzini) [inline src/main/resources/stories/runtime-bundle/task-actions.story]: @constantin-ciobotaru i reported this line also on Diego's PR can you please have a look at this comment and see if it makes sense to you? Thanks.
https://github.com/Activiti/activiti-cloud-acceptance-tests/pull/1#issuecomment-365982825

File: src/main/java/org/activiti/cloud/qa/story/ProcessInstanceTasks.java [hunk 2]
```diff
@@ -79,4 +85,13 @@ public void verifyProcessStatus() throws Exception {
         assertThat(resultingEvent.getEventType()).isEqualTo(EventType.TASK_COMPLETED);
         assertThat(resultingEvent.getTask().getId()).isEqualTo(currentTask.getId());
     }
+
+    @Then("the status of the process is changed to completed by querying")
```
Comment (ffazzini) [inline src/main/java/org/activiti/cloud/qa/story/ProcessInstanceTasks.java]: If we remove the last line from the user story file, this step definition implementation should be moved into "the status of the process is changed to completed"

File: src/main/resources/stories/runtime-bundle/task-actions.story (hunk 1 above)
Comment (constantin-ciobotaru) [reply src/main/resources/stories/runtime-bundle/task-actions.story]: @ffazzini Agree, I'll update the PR.

**Pull request merged**